outputs.tf

```
@@ -0,0 +1,65 @@
+################################################################################
+# mongodbatlas_project
+################################################################################
+
+output "project_id" {
+  value = mongodbatlas_project.project.id
+}
+
+output "creation_timestamp" {
+  value = mongodbatlas_project.project.created
+}
+
+################################################################################
+# mongodbatlas_project_ip_access_list
+################################################################################
+
+output "ips_id" {
```

```suggestion
output "ip_ids" {
```
WDYT? same for `cidr_ids`

---

Sounds better!